crates/bootstrap_srv/src/bin/kitsune2-bootstrap-srv.rs

```diff
@@ -5,19 +5,27 @@ use kitsune2_bootstrap_srv::*;
 #[derive(clap::Parser, Debug)]
 #[command(version)]
 pub struct Args {
-    /// By default kitsune2-boot-srv runs in "testing" configuration
+    /// By default, kitsune2-boot-srv runs in "testing" configuration
     /// with much lighter resource usage settings. This testing mode
     /// should be more than enough for most developer application testing
     /// and continuous integration or automated tests.
     ///
-    /// To setup the server to be ready to use most of the resources available
+    /// To set up the server to be ready to use most of the resources available
     /// on a single given machine, you can set this "production" mode.
     #[arg(long)]
     pub production: bool,
+
     /// Output tracing in json format.
     #[arg(long)]
     pub json: bool,
-    // TODO - Implement the ability to specify TLS certificates
+
+    /// The path to a TLS certificate file.
+    #[arg(long, requires = "tls_key")]
```

Comment:
Try to be helpful when specifying these on the command line, they cannot be specified separately

Reply:
oo, that's nice :+1: